src/main/xerus/monstercat/api/response/Release.kt

```diff
@@ -41,6 +43,17 @@ data class Release(
 		return this
 	}
 	
+	/** Updates every track, linking it to its release.
+	 * Can be used after a track fetch.
+	 * [tracks] Tracks corresponding to that release which needs to be linked.
+	 */
+	fun updateTracks(tracks: List<Track>) {
```

Comment:
what is this for?

Reply:
Tracks fetched from the cache are not linked to Releases in the init. This is a workaround to force-link the tracks and the releases.